app/utils/Strings.scala

```diff
@@ -38,6 +38,25 @@ object Strings {
     Random.alphanumeric.filter(_.isDigit).take(length).mkString
   }
 
+  def decimal(length: Int): String =
```

Comment:
Not to be awkward, but how is this different to numeric now?

Reply:
It might be best to change the name back to decimalMax12. It's unfortunate but there are similar names in the regexes where the regex is completely different in implementation.